Library/Homebrew/cask/lib/hbc/cli/internal_stanza.rb

```diff
@@ -67,7 +67,7 @@ def run
 
           if value.nil? || (value.respond_to?(:to_a) && value.to_a.empty?) ||
              (value.respond_to?(:to_s) && value.to_s == "{}") ||
-             (artifact_name && !value.key?(artifact_name))
+             (artifact_name && !value.include?(artifact_name))
```

Comment:
Pretty sure that `value` will never include `artifact_name`. Did you check what is inside `value`?

Reply:
Here is some example output of "value" when running "brew cask _stanza artifacts ableton-live" https://gist.github.com/Git-Jiro/6ee0421cf4ab1d59f008251760fac3a3 ...

Reply:
What if you run `brew cask _stanza app <cask>`?

Reply:
This is the output of "brew cask _stanza app ableton-live" https://gist.github.com/Git-Jiro/66079ce487d44500207988eb54a6a4f2

Reply:
I am still wondering why "brew tests" works on my machine and fails on Travis-CI.
As far as I can tell Travis-CI is using this bottled Ruby to run the tests: https://homebrew.bintray.com/bottles-portable/portable-ruby-2.3.3.leopard_64.bottle.1.tar.gz
As far as I can see my local Homebrew seems to use exactly the same bottled Ruby version: https://gist.github.com/Git-Jiro/30093a888c2c2ac929f195b2457cb6e6

Reply:
You mentioned that inserting a `raise` statement broke `brew cask _stanza`. Did it not break `brew tests`?

Reply:
Yes, the raise breaks "brew tests" as well.